react/SelectBox/Readme.md

```diff
@@ -66,6 +66,28 @@ const options = [
 />
 ```
 
+### Options with actions
+
+You can display additional actions inside an Option with the ActionsOption component.
+
+```
+const ActionsOption = require('../SelectBox').ActionsOption;
+const options = [
+  { value: 'chocolate', label: 'Chocolate' },
+  { value: 'strawberry', label: 'Strawberry', isDisabled: true },
+  { value: 'vanilla', label: 'Vanilla' }
+];
+
+const CustomOption = (props) => (<ActionsOption {...props} actions={[
+    { icon: 'delete', handler: () => alert('deleting') },
```

Comment:
`handler`? Since this is designed to be executed during the `onClick`, why not naming it `onClick` ?

Reply:
That would work too — but it's also the handler for the action, or the event. I don't know, I don't think one is much better than the other ?